utils/cliutils/utils.go

```diff
@@ -137,6 +139,31 @@ func PrintHelpAndReturnError(msg string, context *cli.Context) error {
 	return errors.New(msg)
 }
 
+func GetQuietValue(c *cli.Context) bool {
```

Comment:
I think this logic worth some comment to explain the connection between "CI" and "quiet"